providers/forty-six-elks/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@novu/forty-six-elks",
+  "version": "^0.12.0",
```

Comment:
```suggestion
  "version": "0.12.0",
```

Reply:
✅ Commited the change now, assumed the auto-generated value was a syntax I didn't know and was correct :D.